client/src/components/Posts.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { useEffect, useState } from 'react'
+import { BrowserRouter, Link, Route, Switch } from 'react-router-dom'
```

Comment:
I noticed `BrowserRouter, Route & Switch` were brought in here but never used through various alerts in the browser and terminal consoles. It is good practice to take a pass through and remove any assets that were not actually needed just like cleaning up comments and unused functionality.